Projects/Calendar/index.html

```diff
@@ -1,12 +1,10 @@
 <!DOCTYPE html>
-<!-- Coding By CodingNepal - youtube.com/codingnepal -->
 <html lang="en" dir="ltr">
   <head>
     <meta charset="utf-8" />
     <title>Dynamic Calendar JavaScript | CodingNepal</title>
```

Comment:
hey @khushigarg1 Change the title of the project

Reply:
Done!